flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/filesystem/FileSystemOptions.java

```diff
@@ -88,4 +88,48 @@
 							" If timestamp in partition is year, month, day, hour," +
 							" can configure: '$year-$month-$day $hour:00:00'." +
 							" If timestamp in partition is dt and hour, can configure: '$dt $hour:00:00'.");
+
+	public static final ConfigOption<String> SINK_PARTITION_COMMIT_TRIGGER =
+			key("sink.partition-commit.trigger")
+					.stringType()
+					.defaultValue("partition-time")
+					.withDescription("Trigger type for partition commit:" +
+							" 'partition-time': extract time from partition," +
+							" if 'watermark' > 'partition-time' + 'delay', will commit the partition." +
+							" 'process-time': use processing time, if 'current processing time' > " +
+							"'partition creation time' + 'delay', will commit the partition.");
+
+	public static final ConfigOption<Duration> SINK_PARTITION_COMMIT_DELAY =
+			key("sink.partition-commit.delay")
+					.durationType()
+					.defaultValue(Duration.ofMillis(0))
+					.withDescription("The partition will not commit until the delay time." +
+							" if it is a day partition, should be '1 d'," +
+							" if it is a hour partition, should be '1 h'");
+
+	public static final ConfigOption<String> SINK_PARTITION_COMMIT_POLICY_KIND =
+			key("sink.partition-commit.policy.kind")
+					.stringType()
+					.noDefaultValue()
+					.withDescription("Policy to commit a partition is to notify the downstream" +
+							" application that the partition has finished writing, the partition" +
+							" is ready to be read." +
+							" metastore: add partition to metastore." +
+							" success-file: add '_success' file to directory." +
+							" Both can be configured at the same time: 'metastore,success-file'." +
+							" custom: use policy class to create a commit policy." +
+							" Support to configure multiple policies: 'metastore,success-file'.");
+
+	public static final ConfigOption<String> SINK_PARTITION_COMMIT_POLICY_CLASS =
+			key("sink.partition-commit.policy.class")
+					.stringType()
+					.noDefaultValue()
+					.withDescription("The partition commit policy class for implement PartitionCommitPolicy interface.");
```

Comment:
Does this only work custom commit policy? If so it should be mentioned in the description and reflected in the config name.

Reply:
Just add comment is OK, has been mentioned in policy kind.